includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -684,7 +684,7 @@ public function get_payment_fields_js_config() {
 	 */
 	public function is_setup_intent_success_creation_redirection() {
 		return (
-			! empty( $_GET['setup_intent_client_secret'] ) & ! empty( $_GET['setup_intent'] ) & ! empty( $_GET['redirect_status'] ) && 'succeeded' === $_GET['redirect_status'] ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+			! empty( $_GET['setup_intent_client_secret'] ) && ! empty( $_GET['setup_intent'] ) && ! empty( $_GET['redirect_status'] ) && 'succeeded' === $_GET['redirect_status'] ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
This condition would be easier to read in multi-line.

Reply:
Done in 5cd75bc13d3c4fd7c16a30232a7de65a14855925